### ISSUE TITLE
BSE encryption

### ISSUE THREAD

**Issue by minirop** [marcussacana/EthornellEditor#3]
Hello,
someone told me some years ago how BSE encryption works so I added it to my program: [here's the code](https://github.com/minirop/arc-reader/blob/master/bse.c).
regards

**Comment by marcussacana** [comment]
Yeah, I saw some codes but I since I saw many tools to extract BGI files automatically decrypt the BSE, I just think as a waste of time, but I should give a try any time.
